### PR TITLE
How to build HPX with Visual Studio

### PR DESCRIPTION
**Pull request by brakmic**
This little tutorial is based on my experiences with building HPX under Windows 10 & VS2015.  Many thanks to the HPX team for this wonderful library and especially to @biddisco and @hkaiser who patiently supported me during my efforts to build HPX on my machine.

### CODE REVIEW

**Comment by hkaiser** [comment]
@brakmic Could you please add the Boost license and your copyright to your file (see here for how to do that: https://github.com/STEllAR-GROUP/hpx/blob/master/CONTRIBUTING.md and for some rationale)

**Comment by hkaiser** [comment]
@brakmic See here for the inspect report revealing the problem: https://circle-artifacts.com/gh/STEllAR-GROUP/hpx/1326/artifacts/0/tmp/circle-artifacts.IBeoeYz/hpx_inspect_report.html

**Comment by brakmic** [comment]
@hkaiser I've added the boost copyright and fixed the formatting (also found a small typo)

**Comment by sithhell** [comment]
Why was this closed?

**Comment by brakmic** [comment]
I thought that after the successful build the request can be closed and the patch-branch will be later merged.

File: docs/tutorial/win32_build.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+//  Copyright (c) 2015 Harris Brakmic<br/>
+//<br/>
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying<br/>
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)<br/>
+<br/>
+**How to build HPX under Windows 10 x64 with Visual Studio 2015**
+
+* a) Download the *CMake 3.3.0* installer from <a href="http://www.cmake.org/files/v3.3/cmake-3.3.0-win32-x86.exe" target="_blank">here</a> <br/>
+* b) Download *Portable Hardware Locality* from <a href="http://www.open-mpi.org/software/hwloc/v1.11/downloads/hwloc-win64-build-1.11.0.zip">here</a> and unpack it.<br/>
+* c) Download *Boost libraries* from <a href="http://sourceforge.net/projects/boost/files/boost/1.59.0/boost_1_59_0.zip/download">here</a> and upack them.<br/>
```
Comment (sithhell) [inline docs/tutorial/win32_build.md]: There is a typo here: "unpack"

**Comment by hkaiser** [comment]
What would be the suggested way forwad? This text needs to be integrated with our docs. Should we merge it and then integrate things? Or should we wait for the patch to be ready to put things directly into the docs?

**Comment by brakmic** [comment]
Honestly, I don't know because I've never had similar experiences. Committing a small document with a few pictures in it is usually nothing "expensive" and surely not involving a whole compilation that takes more than 2 hours. I didn't want to provoke such complexities by delivering a *non-source-code* file. I'm sorry for the confusion.

**Comment by hkaiser** [comment]
@brakmic you did not cause any confusion, all is well. The question was targeted at the other developers, mainly. Thanks again for your contribution. BTW, would you be able to add the image files to this pull request as well?

**Comment by hkaiser** [comment]
@brakmic you did not cause any confusion, all is well. The question was targeted at the other developers, mainly. Thanks again for your contribution. BTW, would you be able to add the image files to this pull request as well?

**Comment by brakmic** [comment]
@hkaiser I've added the screenshots and put everything together in a subdir called "win32".

**Comment by brakmic** [comment]
Does anyone know why this test is failing? I've only uploaded a few images.

Thanks!

**Comment by hkaiser** [comment]
@brakmic I'm slowly coming around to have time integrating this with our docs. Do you want to update something based on your recent blog posts before I start merging it in?

**Comment by brakmic** [comment]
I'd only update the version numbers of Boost and CMake. Will do it immediately.

**Comment by hkaiser** [comment]
@brakmic Thanks!

**Pull request merged**